engine/src/main/java/org/terasology/rendering/cameras/OpenVRStereoCamera.java

```diff
@@ -15,51 +15,56 @@
  */
 package org.terasology.rendering.cameras;
 
+import org.terasology.rendering.openvrprovider.OpenVRProvider;
 import org.lwjgl.opengl.GL11;
 import org.terasology.math.MatrixUtils;
 import org.terasology.math.geom.Matrix4f;
 import org.terasology.math.geom.Vector3f;
 import org.terasology.registry.CoreRegistry;
-import org.terasology.rendering.oculusVr.OculusVrHelper;
 import org.terasology.rendering.world.WorldRenderer;
 import org.terasology.rendering.world.WorldRenderer.RenderingStage;
 
 import static org.lwjgl.opengl.GL11.GL_PROJECTION;
 import static org.lwjgl.opengl.GL11.glMatrixMode;
 
 /**
- * Camera which can be used to render stereoscopic images of the scene for the Oculus Rift.
+ * Camera which can be used to render stereoscopic images of the scene for VR.
  *
  */
-public class OculusStereoCamera extends Camera {
+public class OpenVRStereoCamera extends Camera {
```

Comment:
Some info for the future: camera should eventually become components in the Entity System. As the ES is all about reusing and composing components, I see VR cameras as pairs of standard cameras rather than specialized classes. Obviously this will work for the time being. But is there anything in this class that might cause trouble in that vision?